tests/test_mmap.py

```diff
@@ -0,0 +1,83 @@
+import re
+
+import numpy as np
+import pytest
+
+import blosc2
+
+
+@pytest.mark.parametrize("initial_mapping_size", [None, 1000])
+def test_open_offset(tmp_path, monkeypatch, capfd, initial_mapping_size):
+    monkeypatch.setenv("BLOSC_INFO", "true")
+    expected_mapping_size = 1073741824 if initial_mapping_size is None else initial_mapping_size
```

Comment:
Probably 2**30 reads better than 1073741824, but this is in the eye of the beholder.

Reply:
I also think `2**30` reads much nicer :-)